pgmq-extension/sql/pgmq.sql

```diff
@@ -497,9 +502,9 @@ BEGIN
      IF partitioned THEN
         EXECUTE FORMAT(
           $QUERY$
-          DELETE FROM public.part_config where parent_table = '%s'
+          DELETE FROM public.part_config where parent_table in (%L, %L)
           $QUERY$,
-          queue_name
+          fq_qtable, fq_atable
```

Comment:
also part of the above bug, we were using `queue_name` instead of fully qualified table name